### ISSUE TITLE
`save` always updates with server-side state

### ISSUE THREAD

**Issue by remko** [AmpersandJS/ampersand-model#32]
The `save` code says:

>   // After a successful server-side save, the client is (optionally)
>   // updated with the server-side state.

From the code that follows, I was assuming this was controlled using `options.parse`, but it seems i'm wrong. How do you make sure the model does not update itself using the server state?

FYI, I used the following to test this:
```
  // Succeeds
  test("`save` updates model with server response", function (t) {
        t.plan(2);
        var model = new Backbone.Model({props: {testing: 'string'}});
        model.sync = function (method, model, options) { options.success({testing: 'bar'}); };
        model.save({testing: 'foo'}, { parse: true });
        t.equal(model.get('testing'), 'bar');
        t.ok(true);
   });

  // Fails
  test("`save` does not update model with server response", function (t) {
        t.plan(2);
        var model = new Backbone.Model({props: {testing: 'string'}});
        model.sync = function (method, model, options) { options.success({testing: 'bar'}); };
        model.save({testing: 'foo'}, { parse: false });
        t.equal(model.get('testing'), 'foo');
        t.ok(true);
   });
```

**Comment by kamilogorek** [comment]
`{ parse: false }` will only prevent piping data through model's parse function.

> How do you make sure the model does not update itself using the server state?
Why would you exactly want to do that?

**Comment by remko** [comment]
My initial problem was after doing a second change when the first change wasn't confirmed yet; in that case, the response to the first change sets the state to an outdated change. I guess the problem resolves itself eventually when the response to the second change comes in.
In the end, I found out my issue was more fundamental because i'm using a multi-threaded server, so I had to solve it in a different way than avoiding to update state on the client side (which was something i didn't need).
